jscomp/bsb/bsb_merlin_gen.ml

```diff
@@ -152,12 +173,30 @@ let merlin_file_gen ~cwd
         Buffer.add_string buffer merlin_b;
         Buffer.add_string buffer path ;
       );      
+
+#if BS_NATIVE then
+    let nested = match backend with
+      | Bsb_config_types.Js       -> Literals.js
+      | Bsb_config_types.Native   -> Literals.native
+      | Bsb_config_types.Bytecode -> Literals.bytecode
+    in
+#end
```

Comment:
This is the same thing, we can push it into lazy evaluation since it only needs to be evaluated once during the lifetime of such process

Reply:
Now that it doesn't take any args and it still has a side effect, do you think I should use lazy? It might make sense, I'm just scared of having the issue of calling this too early, because the backend is parsed and be stuck with that wrong value.